src/environments/ui.ts

```diff
@@ -0,0 +1,462 @@
+/* eslint-disable */
+import { environment } from './environment';
```

Comment:
1. We have scripts folder. It may be more appropriate for this file instead of `environments`.
2. We should remove `setup_env.js`.

Reply:
Done and done